marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -40,6 +41,39 @@ import Marlowe.Slot (secondsDiff, slotToDateTime)
 import Material.Icons (Icon(..), icon)
 import WalletData.State (adaToken, getAda)
 
+-- I'm moving this view here so that we can easily call Contract.Actions from inside it. I'm not
+-- actually calling any Contract.Actions from here yet, but that's a TODO for the next PR...
+contractInnerBox :: forall p. Slot -> State -> HTML p Action
+contractInnerBox currentSlot state =
```

Comment:
This isn't the whole box (hence "inner"). I left the parent div and the header in the dashboard view, because that needs to have a clickable element that opens the screen for that contract - which is a dashboard action. But then the rest goes here in the contract view, because it will have contract actions.